### DISCUSSION TITLE
Question: any way to detect mail inbox drag and drop.

### DISCUSSION THREAD

**Discussion by vishnug-bridge** [InboxSDK/InboxSDK#1196] [Q&A]
I am creating drag-and-drop functionality inside Gmail. I want to detect when an email is dropped. Is there any way to detect the drag and drop?

![Screenshot from 2024-05-13 21-00-33](https://github.com/InboxSDK/InboxSDK/assets/147032947/23e8e287-2f52-4575-a316-6c104067edb1)

**Comment by wegry** [comment]
`@inboxsdk/core` doesn't currently support this, but we'd be open to PRs.
